Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java

```diff
@@ -155,7 +172,7 @@ public void onDestroyView()
   {
     super.onDestroyView();
 
-    NativeLibrary.SetObscuredPixelsLeft(0);
+    NativeLibrary.SetObscuredPixelsLeft(mCutInset);
```

Comment:
Should this also be applied when EmulationActivity/EmulationFragment is created? (In the case where the user doesn't open the menu)

Reply:
Applied them in the Emulation Activity insets listener